starfish/core/image/_filter/reduce.py

```diff
@@ -17,53 +22,141 @@
 
 class Reduce(FilterAlgorithmBase):
```

Comment:
@ttung given how codebases typically mirror map & reduce, I wonder if `apply` should be exposed in a similar fashion.

Reply:
Yes, I agree.  I thought there was already an issue for that, but maybe I'm wrong.

It's also possible that Reduce might work for this already.